skimage/color/colorconv.py

```diff
@@ -1516,10 +1526,12 @@ def combine_stains(stains, conv_matrix):
     from ..exposure import rescale_intensity
 
     stains = _prepare_colorarray(stains)
-    logrgb2 = -np.reshape(stains, (-1, 3)) @ conv_matrix
-    rgb2 = np.power(10, logrgb2)
-    return rescale_intensity(np.reshape(rgb2 - 2, stains.shape),
-                             in_range=(-1, 1))
+    # log_adjust here is used to compensate the sum within separate_stains().
+    log_adjust = -np.log(1E-6)
+    log_rgb = -(stains * log_adjust) @ conv_matrix
```

Comment:
```suggestion
    log_adjust = np.log(1E-6)
    log_rgb = (stains * log_adjust) @ conv_matrix
```
Can we remove this double negation?